konig-core/src/main/java/io/konig/shacl/Shape.java

```diff
@@ -88,7 +88,8 @@
 
 	private URI aggregationOf;
 	private URI rollUpBy;
-	private String mediaTypeBaseName;
+	private String mediaTypeBaseName;
+	private URI shapeType;

```

Comment:
Remove the `shapeType` field.  Use the exisiting `List<URI> type` field instead.  (A Shape may have more than one `type` value.)